pkg/connector/group.go

```diff
@@ -247,6 +255,9 @@ func (o *groupResourceType) Grants(
 			}
 			usersCount, ok := sdkResource.GetProfileInt64Value(groupTrait.Profile, usersCountProfileKey)
 			shouldExpand := !ok || usersCount > 0
+			if !shouldExpand {
+				l.Info("okta-connectorv2: skipping expand for role group grant since users_count is 0")
```

Comment:
Should these l.Info()s be debugs? I think they'd be pretty spammy on large Okta accounts.